### PR TITLE
create Dockerfile

### PR DESCRIPTION
**Pull request by Derrick56007**

### CODE REVIEW

**Comment by jespino** [comment]
Thanks @Derrick56007 we need you to sign the CLA to accept your PR, anway, somebody already submitted another PR with a preliminary version of the Dockerfile, so if you want to apply changes to that docker file would be really useful.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ENV DEBIAN_FRONTEND=noninteractive
+
+ENV GOLANG_VERSION="1.16.2"
+
+ENV NODE_VERSION="15"
+
+# Install needed packages and clean up.
+RUN apt-get update \
+	&& apt-get install -y curl wget git make gcc g++ ca-certificates --no-install-recommends \
+    # Go installation
```
Comment (obbardc) [inline Dockerfile:15]: be consistant with spaces vs. tabs

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+ENV GOLANG_VERSION="1.16.2"
+
+ENV NODE_VERSION="15"
+
+# Install needed packages and clean up.
+RUN apt-get update \
+	&& apt-get install -y curl wget git make gcc g++ ca-certificates --no-install-recommends \
```
Comment (obbardc) [inline Dockerfile:14]: put apt arguments before pacakge names, then sort packages one per line in A-Z order

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    && curl -fsSL https://deb.nodesource.com/setup_${NODE_VERSION}.x | bash - \
+    && apt-get install -y nodejs --no-install-recommends
+
+ENV PATH "$PATH:/usr/local/go/bin"
+
+RUN  git clone https://github.com/mattermost/focalboard.git
+
+WORKDIR /focalboard
+
+RUN make prebuild && make \
```
Comment (obbardc) [inline Dockerfile:29]: new line per command, keep the && on the previous line

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:bionic
```
Comment (obbardc) [inline Dockerfile:1]: add to README how to build and run the docker container

**Review by obbardc — CHANGES_REQUESTED** [review]

**Comment by obbardc** [comment]
Huh, i completely missed there is already a dockerfile in the main repo. Guess this can be closed

**Comment by it33** [comment]
@obbardc thanks for taking the initiative! Are there other ideas or projects you'd be interested in around focalboard?

**Comment by proffalken** [comment]
> Huh, i completely missed there is already a dockerfile in the main repo. Guess this can be closed

Heh, great minds and all that, I put in a basic one yesterday, completely missed this PR!

The one I created definitely needs some love though, so please make changes as you see fit!

**Comment by jasonblais** [comment]
Thanks for the contribution!

@jespino is this a duplicate of https://github.com/mattermost/focalboard/pull/76?

**Comment by jespino** [comment]
Yes, this can be closed, we already have a docker file and actually @flexo3001 is already improving it in another PR.

**Pull request closed without merging**